src/NuGet.Indexing/NuGetSearcherManager.cs

```diff
@@ -206,7 +207,7 @@ protected override NuGetIndexSearcher CreateSearcher(IndexReader reader)
                 }
                 catch (Exception e)
                 {
-                    _logger.LogError("NuGetSearcherManager.CreateSearcher: Error loading auxiliary data.", e);
+                    _logger.LogError("NuGetSearcherManager.CreateSearcher - Exception thrown while loading auxiliary data: {Exception}", e);
```

Comment:
if `verifiedPackages.json` is not found would this exception not reload any aux data? I am worried what would happen if we deployed search service before the job is deployed.